src/rustllvm/PassWrapper.cpp

```diff
@@ -97,6 +97,55 @@ LLVMRustAddPass(LLVMPassManagerRef PM, Pass *pass) {
     pm->add(pass);
 }
 
+#define GEN_SUBTARGETS \
+    SUBTARGET(X86) \
+    SUBTARGET(ARM) \
+    SUBTARGET(AArch64) \
+    SUBTARGET(Mips) \
+    SUBTARGET(PPC)
```

Comment:
Let's avoid having *too* much duplication, but otherwise yes I suspect something similar will have to happen here